package.json

```diff
@@ -48,9 +48,9 @@
   },
   "devDependencies": {
     "@angular-builders/jest": "~7.1.2",
-    "@angular-devkit/build-angular": "~0.11.4",
+    "@angular-devkit/build-angular": "^0.13.8",
```

Comment:
Please undo these unnecessary changes to package.json.

Reply:
I do not recall making that change, but I believe it updated itself as I had to run npm install again, and updated the system since l I was having trouble building it